docs/api/proposed_endpoints.md

```diff
@@ -356,17 +356,20 @@ Name               | Data type     | Description
 -------------------|---------------|---------------------
 id                 | UUID          | Unique ID for the proposal
 title              | text          | Title of proposal
-speaker            | text          | Speaker for the talk
-kind               | text          | Type of proposal like talk, dev sprint, workshop
+speaker            | UUID          | Foreign Key to `User` Model
```

Comment:
The client doesn't care if you're using SQL and have a foreign key. It is just `user_id` for the user :)